docs/orchestrating-elastic-stack-applications/elasticsearch/jvm-heap-dumps.asciidoc

```diff
@@ -0,0 +1,44 @@
+:parent_page_id: elasticsearch-specification
+:page_id: jvm-heap-dumps
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{parent_page_id}.html#k8s-{page_id}[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= JVM heap dumps
+
+== Ensure sufficient storage
+Elasticsearch is link:https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html#heap-dump-path[configured by default] to take heap dumps on out of memory exceptions to the default data directory. The default data directory is `/usr/share/elasticsearch/data` in the official Docker images ECK uses. If you are running Elasticsearch with a large heap that is as large as the remaining space on the data volume this can lead to a situation where Elasticsearch is no longer able to start. To avoid this scenario you have two options:
```

Comment:
```suggestion
Elasticsearch is link:https://www.elastic.co/guide/en/elasticsearch/reference/current/important-settings.html#heap-dump-path[configured by default] to take heap dumps on out-of-memory exceptions to the default data directory. The default data directory is `/usr/share/elasticsearch/data` in the official Docker images that ECK uses. If you are running Elasticsearch with a large heap that is as large as the remaining space on the data volume, this can lead to a situation where Elasticsearch is no longer able to start. To avoid this scenario you have two options:
```